### PR TITLE
Thu/adapt getembedinfo for keycloak tokens

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: doc/powerBI.md [hunk 1]
```diff
@@ -620,40 +620,90 @@ The webapp currently supports **two different modes to retrieve the PowerBI toke
...
+dashboards. It requires to have an existing Azure app registration, whose credentials (client id and client secret) will
+be used to retrieve an access token. **This app registration must be added as a "_Member_" of your PowerBI workspace**.
+
+For security reasons, it is recommended to create a different app registration for this service account, and not to
+reuse the existing app registration associated to your Azure Static Web App.
+
+The configuration of the [`GetEmbedInfo`](apiAzureFunctions.md) Azure Function can be done from your Static Web App
+instance. Open the "_Configuration_" blade
+(_Azure Portal_ > _Static Web Apps_ > _[name of your webapp]_ > _Configuration_), and add the environment variables
+below:
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply doc/powerBI.md:704]: Fixed, thank you :slightly_smiling_face:

File: api/GetEmbedInfo/authentication.js [hunk 2]
```diff
@@ -13,14 +13,14 @@ if (MSAL_CONFIG?.auth?.clientId && MSAL_CONFIG?.auth?.clientSecret) {
 // With client credentials flows permissions need to be granted in the portal by a tenant administrator.
 // The scope is always in the format "<resource>/.default"
 const clientCredentialRequest = {
-  scopes: [getConfigValue('POWER_BI_SCOPE')],
+  scopes: ['https://analysis.windows.net/powerbi/api/.default'],
```
Comment (nborde-CSM) [inline api/GetEmbedInfo/authentication.js:16]: Maybe set this url in a lone const ?

**Review by nborde-CSM — APPROVED** [review]
One remark but LGTM :+1:

File: api/GetEmbedInfo/authentication.js (hunk 2 above)
Comment (csm-thu) [reply api/GetEmbedInfo/authentication.js]: Done :+1:

**Pull request merged**